src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher2.php

```diff
@@ -35,7 +37,7 @@ public function match($pathinfo)
         if (0 === strpos($pathinfo, '/bar')) {
             // bar
             if (preg_match('#^/bar/(?P<foo>[^/]++)$#s', $pathinfo, $matches)) {
-                if (!in_array($this->context->getMethod(), array('GET', 'HEAD'))) {
+                if (!in_array($requestMethod, array('GET', 'HEAD'))) {
```

Comment:
Sorry, I wasn't clear enough. My idea was to create a specific variable (different from `$requestMethod`, like `$isLikeGetMethod` or something) that holds `GET` when the real method is `HEAD`.